src/partials/nav-version-control.hbs

```diff
@@ -16,13 +16,13 @@
                                   <li{{#if (eq ./version @root.page.version)}} class="current"{{/if}}><a class="frame-item" href="{{relativize ./url}}">Version {{./displayVersion}}</a></li>
                               {{/each}}
                               </ul>
-                              {{#if (ends-with page.component.name '-sdk')}}
+                              {{!-- {{#if (ends-with page.component.name '-sdk')}}
```

Comment:
Please remove old code instead of commenting out when submitting a PR.

Reply:
Ok, Sure. I"ll remove all commented code and console things as well.